model/src/main/java/org/activityinfo/model/formTree/LookupKey.java

```diff
@@ -233,4 +270,14 @@ public String label(FormInstance record) {
     public String getKeyLabel() {
         return keyLabel;
     }
-}
+
+    public void addParentLevel(ParentKey parent) {
```

Comment:
Speaking of access modifiers, this could probably be package private, as this should only be called when constructing the key graph.

Reply:
Yes, and in fact it's not currently called as we actually just pass in the list of parent keys we've constructed in LookupKeySet when we create a new LookupKey. I'll remove the modifier so its `package-private`.